.github/workflows/release-crypto-icons.yml

```
@@ -0,0 +1,44 @@
+name: "[Release] crypto icons"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs at midnight UTC every Monday
+    - cron: "0 0 * * *"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release-crypto-icons:
+    environment: ${{ github.event.inputs.environment }}
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: "eu-central-1"
+      AWS_CLOUDFRONT_ID: E1ERY5K2OTKKI1
+      COINGECKO_API_KEY: ${{ secrets.COINGECKO_API_KEY }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          # role-to-assume: # TODO:
```

this should be set to
```suggestion
          role-to-assume: arn:aws:iam::538326561891:role/gh_actions_suite_production_icons
```

---

fixed in https://github.com/trezor/trezor-suite/pull/12694/commits/8489b3c8d033ea65d640e0e8d9a361864c24ab55